queries/places.jinja2

```diff
@@ -70,9 +74,13 @@ SELECT
     NULL AS population,
     NULL AS capital,
     NULL AS state_capital,
+    n_photos,
```

Comment:
the `kind_tile_rank` property should already allow clients to sort / collide / filter neighborhood features, and is based on `n_photos`. Let's only expose this to verify during dev, but remove before taking it to prod. It's not useful to general consumers (but `transit_lines` is in the other issue).